src/android/Fingerprint.java

```diff
@@ -20,11 +20,13 @@
 public class Fingerprint extends CordovaPlugin {
 
     private static final String TAG = "Fingerprint";
-    private CallbackContext mCallbackContext = null;
-
     private static final int REQUEST_CODE_BIOMETRIC = 1;
+
+    private CallbackContext mCallbackContext = null;
     private PromptInfo.Builder mPromptInfoBuilder;
 
+    static final String SECRET_EXTRA = "secret";
```

Comment:
A paramter for this as well?

Reply:
It is the key of the map, no point to have a customer value for that.
Not a good practice to have Fingerprint.java using BiometricActivity.java and BiometricActivity.java using Fingerprint.java

Would make sense to move it to PromptInfo.java